odm2api/ODM2/services/readService.py

```diff
@@ -544,12 +544,30 @@ def getPeople(self, ids=None, firstname=None, lastname=None):
 
     def getAffiliations(self, ids=None, personfirst=None, personlast=None, orgcode=None):
         """
-        getAffiliations(self, ids=None, personfirst=None, personlast=None, orgcode=None)
-        * Pass nothing - returns a list of all Affiliation objects
-        * Pass a list of AffiliationID - returns a single Affiliation object
-        * Pass a First Name - returns a single Affiliation object
-        * Pass a Last Name - returns a single Affiliation object
-        * Pass an OrganizationCode - returns a Affiliation object
+        Retrieve a list of ODM2 Affiliation objects.
+
+        Parameters
+        ----------
+        ids: list, default None
+            List of AffiliationIDs
+        personfirst: str, default None
+            Person First Name
+        personlast: str, default None
+            Person Last Name
+        orgcode: str, default None
+            Organization Code
+
```

Comment:
Looks like if you call the function with no arguments (or None for all / any of them), a list of **ALL** affiliations will be returned. Tweak the docstring to include that information.

Reply:
Should I say:

```
Retrieve a list of ODM2 Affiliation objects.
        
Note: If no arguments are passed to the function, 
ALL ODM2 Affiliation objects within the database will be queried.
```

Reply:
```
Retrieve a list of Affiliation objects.
        
If no arguments are passed to the function, or their values are None, 
all Affiliation objects in the database will be returned.
```